src/tx/builder/schema.js

```diff
@@ -752,8 +752,8 @@ const CHANNEL_FORCE_PROGRESS_TX = [
   TX_FIELD('fromId', FIELD_TYPES.id, 'ak'),
   TX_FIELD('payload', FIELD_TYPES.binary, 'tx'),
   TX_FIELD('round', FIELD_TYPES.int),
-  TX_FIELD('update', FIELD_TYPES.binary),
-  TX_FIELD('stateHash', FIELD_TYPES.binary),
+  TX_FIELD('update', FIELD_TYPES.binary, 'cb'),
+  TX_FIELD('stateHash', FIELD_TYPES.binary, 'st'),
```

Comment:
Ensured that this is correct by requesting it from middleware 😃 
https://mainnet.aeternity.io/mdw/txs/backward?type=channel_force_progress&limit=10&page=1